packages/eventual-send/src/track-turns.js

```diff
@@ -1,11 +1,12 @@
 /* global globalThis */
+import { X, errorNote } from '@endo/errors';
 import {
   getEnvironmentOption,
   environmentOptionsListHas,
 } from '@endo/env-options';
 
 // NOTE: We can't import these because they're not in scope before lockdown.
-// import { assert, details as X, Fail } from '@agoric/assert';
+// import { assert, X, Fail } from '@agoric/assert';
```

Comment:
Either this is still true, in which case we should back out this change, or it is false, in which case we should remove this comment. If we pass CI with this change, that is weak evidence that this statement is false.

Reply:
CI did fail with
```
Error: Cannot initialize @endo/errors, missing globalThis.assert, import 'ses' before '@endo/errors'
```
which clearly indicates that this comment is still true. Will back out changes to the eventual send package (except the drive-by)

Reply:
Done. Resolving.